### ISSUE TITLE
concourse automation should not depend on dockerhub image latest versions

### ISSUE THREAD

**Issue by poblin-orange** [orange-cloudfoundry/paas-templates#97]
### Expected behavior
As a platform operator, i expect new docker images published on dockerhub should not break my on premises pipeline

### Observed behavior
8/8, concourse pipeline resource dockerhub image for concourse 4 breaks all COA builds.

### Affected release

COA < 3.x

### Traces and logs

* [ ] I have reviewed provided traces against secrets (credentials, internal URLs) that should not be leake, manually of using some tools such as [truffle-hog file:///user/dxa4481/codeprojects/mytraces.txt](https://github.com/dxa4481/truffleHog#truffle-hog)

**Comment by poblin-orange** [comment]
fixed by https://github.com/orange-cloudfoundry/cf-ops-automation/releases/tag/v3.0.0

**Comment by poblin-orange** [comment]
https://success.docker.com/article/planned-downtime-on-hub-cloud-store

**Comment by o-orand** [comment]
With COA [3.0](https://github.com/orange-cloudfoundry/cf-ops-automation/releases/tag/v3.0.0), we've fixed  resource-type docker images, but some task docker images are not using a specific tag.

**Comment by poblin-orange** [comment]
nexus proxy repo set up in v41, activated in v42.
Remaining latest version identified:
- orangecloudfoundry/git-branch-heads-resource
- cfcommunity/bosh2-errand-resource
- cftoolsmiths/cron-resource

**Comment by poblin-orange** [comment]
as of 15/10/2019, another docker hub incident illustrates the internet dependency.

```
failed to fetch digest for image 'orangecloudfoundry/spiff:latest': 503 Service Unavailable
failed to fetch digest for image 'governmentpaas/spruce:b3185eed905c8320013c5d09b6b18cdc0b68ce36': 503 Service Unavailable
does the image exist?

```

**Comment by poblin-orange** [comment]
fixed in 45.0.0 (COA 4.2)
